client/src/app/web-extension/App.js

```diff
@@ -27,6 +28,13 @@ const getRoutes = (routes) => {
 const App = () => {
 	const { mainRoutes, innerRoutes, outerRoutes } = routeConfig;
 
+	useEffect(() => {
+		setPopupOpenState(true);
```

Comment:
This is something I mentioned with bad variable naming, `Popup` is a generic name, it's not clear enough to tell what `popup` specifically does.

For quick resolve, I recommend:
`import { setPopupOpenState as setAuthModalOpenState } from '@cd/components/hooks/useServiceWorker';`

Reply:
It's not authModal a @duc-q , the popup here mean the extension popup  . Let me update the name

Reply:
It's not authModal a @duc-q , the popup here mean the extension popup . Let me update the name